src/runtime/cli/kata-env_test.go

```diff
@@ -272,6 +273,10 @@ VERSION_ID="%s"
 		expectedHostDetails.CPU.Model = "v8"
 	}
 
+	mem := getMemoryInfo()
+	expectedHostDetails.Memory = mem
+	expectedHostDetails.CPU.Cores = runtime.NumCPU()
```

Comment:
NumCPU returns the logical CPUs as visible to the OS. It may not be always equal to the cores. IMHO, naming the variable as Cores is not semantically correct. My suggestion will be using just `CPUs` instead of `Cores`